packages/openneuro-app/src/scripts/client.jsx

```diff
@@ -31,8 +30,6 @@ loadConfig().then(config => {
     })
   }
 
-  GoogleAnalytics.initialize(config.analytics.trackingId)
```

Comment:
This part still needs to happen so it can get the GA_MEASUREMENT_ID from configuration.

```ts
gtag('config', config.analytics.trackingId);
```